lib/github_notification_monitor.rb

```diff
@@ -10,6 +10,7 @@ class GithubNotificationMonitor
     "remove_label"  => :remove_labels,
     "rm_label"      => :remove_labels,
     "assign"        => :assign,
+    "add_reviewer"  => :review,
```

Comment:
Shouldn't the value also be `:add_reviewer` to allow for `:rm_reviewer` in the future?